### PR TITLE
Add FAQ entry about organisation permissions

### LINKED ISSUE #220: Not Receiving Notifications
Hi,

I installed gitify and set up my account but I'm still not receiving any notifications.  Please see my screenshot,
screen shot 2016-09-12 at 10 55 26 am

Thanks,
Chris

### PR DESCRIPTION
**Pull request by simonsmith**
This fixes #220

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -75,6 +75,16 @@ There are 2 linters for `js` & `scss` and unit tests with `mocha`.
     // Run unit tests with coverage
     npm run coverage
 
+### FAQ
+
+#### My notifications aren't showing?
+
+Some organisations require applications to request access before allowing
+access to any data (including notifications) about their repositories.
```
Comment (manosim) [inline README.md]: Can you avoid the breakline please?

File: README.md [hunk 2]
```diff
@@ -75,6 +75,16 @@ There are 2 linters for `js` & `scss` and unit tests with `mocha`.
...
+
+To check if Gitify is approved by your organisation you can go to
+https://github.com/settings/applications, then click on **Gitify** and scroll to
```
Comment (manosim) [inline README.md]: Same as above (applies to next line too).

**Comment by simonsmith** [comment]
My editor default crept in there :). Updated

**Comment by manosim** [comment]
Amazing! Thank you for contributing! Will merge it now 🎉

**Pull request merged**